feature-libs/product-configurator/rulebased/components/config/configurator-ui-config.ts

```diff
@@ -0,0 +1,13 @@
+export abstract class ConfiguratorUIConfig {
+  rulebasedConfigurator: {
```

Comment:
As we are dealing with a configurator wouldn't it be better to use the term "settings" so that there are no confusions with "Configurator" <-> "Config". Suggestion: ConfiguratorUiSettings